hopsworks-common/src/main/java/io/hops/hopsworks/common/dao/AbstractFacade.java

```diff
@@ -17,8 +17,8 @@ public void save(T entity) {
     getEntityManager().persist(entity);
   }
 
-  public void update(T entity) {
-    getEntityManager().merge(entity);
+  public T update(T entity) {
```

Comment:
@berthoug What's the motivation for changing the return type?

Reply:
Because I needed to get the state of the object after the update/merge and because it mirror better the behaviour of the javax.persistence.merge which is used internally in the update functions.